sklearn/linear_model/_glm/glm.py

```diff
@@ -136,8 +142,8 @@ def __init__(
         *,
         alpha=1.0,
         fit_intercept=True,
-        family="normal",
-        link="auto",
+        base_loss_class=HalfSquaredError,
```

Comment:
If these parameters are private, should we name them with a leading underscore `_base_loss_class` ?